delete_photos.js

```diff
@@ -2,7 +2,8 @@
 const ELEMENT_SELECTORS = {
     checkboxClass: '.ckGgle',
     deleteButton: 'button[title="Delete"]',
-    confirmationButton: '#yDmH0d > div.llhEMd.iWO5td > div > div.g3VIld.V639qd.bvQPzd.oEOLpc.Up8vH.J9Nfi.A9Uzve.iWO5td > div.XfpsVe.J9fJmf > button.VfPpkd-LgbsSe.VfPpkd-LgbsSe-OWXEXe-k8QpJ.nCP5yc.DuMIQc.kHssdc.HvOprf'
+    //confirmationButton: '#yDmH0d > div.llhEMd.iWO5td > div > div.g3VIld.V639qd.bvQPzd.oEOLpc.Up8vH.J9Nfi.A9Uzve.iWO5td > div.XfpsVe.J9fJmf > button.VfPpkd-LgbsSe.VfPpkd-LgbsSe-OWXEXe-k8QpJ.nCP5yc.DuMIQc.kHssdc.HvOprf'
```

Comment:
Comments should be avoided in the production code.
We have git to look for the history. Could you remove the commented line?

Reply:
This change has been merged from a different PR: https://github.com/mrishab/google-photos-delete-tool/pull/4